devscripts/generate_fake_sessions.py

```diff
@@ -0,0 +1,97 @@
+"""
```

Comment:
We already have utilities for generating fake user data in https://github.com/learningequality/kolibri/blob/develop/kolibri/core/logger/utils/user_data.py - so I'd rather not add a new script here (I realize that this script is focused on quickly adding the large number of rows for testing, so has a slightly different purpose).

Rather than hold up this PR asking for updates to those utilities (which need a lot of attention), let's just skip committing this script.